spring-core/src/main/java/org/springframework/util/CollectionUtils.java

```diff
@@ -102,10 +104,34 @@ public static <K, V> HashMap<K, V> newHashMap(int expectedSize) {
 	 * @see #newHashMap(int)
 	 */
 	public static <K, V> LinkedHashMap<K, V> newLinkedHashMap(int expectedSize) {
-		return new LinkedHashMap<>(computeMapInitialCapacity(expectedSize), DEFAULT_LOAD_FACTOR);
+		return new LinkedHashMap<>(computeInitialCapacity(expectedSize), DEFAULT_LOAD_FACTOR);
 	}
 
-	private static int computeMapInitialCapacity(int expectedSize) {
+	/**
+	 * Instantiate a new {@link HashSet} with an initial capacity
+	 * that can accommodate the specified number of elements without
+	 * any immediate resize/rehash operations to be expected.
+	 * @param expectedSize the expected number of elements (with a corresponding
+	 * capacity to be derived so that no resize/rehash operations are needed)
+	 * @see #newLinkedHashSet(int)
+	 */
```

Comment:
In the future, please remember to add `@since` tags to newly introduced non-private` methods and to update the copyright headers of all affected files.

Though no need to do that for this PR: I'll take care of that when merging.